frontend/src/lib/components/MoodItem.svelte

```diff
@@ -14,14 +14,14 @@
 
     let moodElement = document.getElementById(`moodElement${id}`);
     if (storedMoodId == 0) {
-      moodElement.classList.add("bg-gray-400");
+      moodElement.classList.add("bg-[#fdefc7]");
```

Comment:
Please do not use colors like that. Add them to the tailwind configuration following the convention